packages/resource-schemas/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@carbon-platform/resource-schemas",
```

Comment:
Thoughts on shorting this package name to `@carbon-platform/schemas`? So in a future state once we update the schema file name, we should shorten the comment to this?

```yml
# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas/library.schema.json
```

Reply:
Done!